src/esw/auton_led.py

```diff
@@ -60,10 +45,27 @@ def main():
 
     rospy.Service("change_auton_led_state", ChangeAutonLEDState, handle_change_auton_led_state)
 
+    color = ""
+    green_counter_s = 0
     seconds_to_wait = 1
```

Comment:
Can change to use seconds_to_wait constant below. May be best to call this variable `period`, make it 2 seconds. Then we could reset `green_counter_s = 0` when `green_counter_s == period`, and flip flop it when `green_counter_s == period/2

Reply:
i changed it to something like that